plugin-packs/postcss-preset-env/src/lib/shared-options.mjs

```diff
@@ -1,11 +1,7 @@
 export function initializeSharedOptions(opts) {
 	if ('preserve' in opts) {
 		const sharedOptions = {};
-
-		if ('preserve' in opts) {
-			sharedOptions.preserve = opts.preserve;
-		}
-
```

Comment:
I don't think this was actually needed?

Reply:
Correct, this function had more parts when `importFrom` and `exportTo` still existed.

This simplified version is better for now 🎉